client/gettext/json/available_languages.json

```diff
@@ -1 +1 @@
-{"es-ES":"Español","fr":"Français","ja":"日本語","ko":"한국어","nl":"Nederlands","pt-BR":"Português do Brasil  ","pt-PT":"Português","zh-Hans-CN":"简体中文","zh-TW":"國語"}
```

Comment:
@mzazrivec  I'm a little concerned about the missing language names for pt-PT zh-TW, ko and nl... if we don't have values specified here, the user won't be able to select any of those languages

Reply:
What I used to do was if the catalog did not have `locale_name` translated in Zanata, I entered it there myself.

Reply:
Yeah that's what I did for the ones that were absent above